### PR TITLE
Package telegraml.2.2.1

### PR DESCRIPTION
**Pull request by OhadRau**
### `telegraml.2.2.1`

Telegram Bot API for OCaml
An implementation of the Telegram Bot API in 100% OCaml,
using Lwt for asynchronous operations and Cohttp for networking.
Implements most basic functionality present in the API, plus
convenience modules for error handling and defining commands.

---
* Homepage: https://github.com/nv-vn/telegraml
* Source repo: https://github.com/nv-vn/telegraml.git
* Bug tracker: https://github.com/nv-vn/telegraml/issues

---
### opam-lint failures
- **WARNING** 27 No field 'remove' while a field 'install' is present, uncomplete uninstallation suspected

---

:camel: Pull-request generated by opam-publish v0.3.5

### CODE REVIEW

**Comment by OhadRau** [comment]
Wondering what the protocol is for new OPAM 1.2 packages. Since Debian hasn't progressed to OPAM 2.0, I've been trying to address a user's difficulty installing this library with 4.07.0. Assuming old format packages won't get merged, should I go ahead and recommend reinstalling OPAM manually?

Thanks,
Ohad

**Comment by kit-ty-kate** [comment]
You just need to upgrade your package to opam2 and that'll be good I think.

File: packages/telegraml/telegraml.2.2.1/opam [hunk 1]
```diff
@@ -0,0 +1,17 @@
+opam-version: "1.2"
+maintainer: "nv-vn <nv@cock.li>"
+authors: "nv-vn <nv@cock.li>"
+homepage: "https://github.com/nv-vn/telegraml"
+bug-reports: "https://github.com/nv-vn/telegraml/issues"
+license: "MIT"
+dev-repo: "https://github.com/nv-vn/telegraml.git"
+build: ["dune" "build" "-p" name "-j" jobs "@install"]
```
Comment (kit-ty-kate) [inline packages/telegraml/telegraml.2.2.1/opam]: Also, `@install` is optional

**Comment by kit-ty-kate** [comment]
The upgrade guide is here: https://opam.ocaml.org/doc/Upgrade_guide.html

**Comment by mseri** [comment]
Alternatively you can re-publish the package using `opam-publish 2`. This would do the conversion for you

**Comment by mseri** [comment]
Alternatively you can re-publish the package using `opam-publish 2`. This would do the conversion for you

**Comment by mseri** [comment]
Alternatively you can re-publish the package using `opam-publish 2`. This would do the conversion for you

**Comment by mseri** [comment]
Alternatively you can re-publish the package using `opam-publish 2`. This would do the conversion for you

**Comment by mseri** [comment]
Alternatively you can re-publish the package using `opam-publish 2`. This would do the conversion for you

**Comment by mseri** [comment]
Alternatively you can re-publish the package using `opam-publish 2`. This would do the conversion for you

**Comment by mseri** [comment]
Alternatively you can re-publish the package using `opam-publish 2`. This would do the conversion for you

**Comment by mseri** [comment]
Alternatively you can re-publish the package using `opam-publish 2`. This would do the conversion for you

**Comment by kit-ty-kate** [comment]
Ping @OhadRau

**Comment by OhadRau** [comment]
Sorry for not responding. I'll be closing this issue since the fix in question was only a workaround for old versions of OPAM.

**Pull request closed without merging**